### PR TITLE
v1.7

### PR DESCRIPTION
**Pull request by pmukherj**
changes to support iOS 13 and higher, 
prepping for upcoming additions to SDK

### CODE REVIEW

File: Placenote.framework/_CodeSignature/CodeResources [hunk 1]
```diff
@@ -10,11 +10,11 @@
 		</data>
 		<key>Headers/libPlacenote.h</key>
 		<data>
-		jPXkxCylBGdrwUtg5yfObmqNXpY=
```
Comment (nmathew87) [inline Placenote.framework/_CodeSignature/CodeResources:13]: what is this file? auto generated ?

File: PlacenoteSDK.podspec [hunk 2]
```diff
@@ -17,7 +17,7 @@ Pod::Spec.new do |s|
     s.license           = { :type => 'Placenote EULA', :file => 'LICENSE' }
 
     s.platform          = :ios, '11.0'
-    s.source            = { :git => 'https://github.com/Placenote/PlacenoteSDK-CocoaPod.git', :tag => 'v1.6.12' }
+    s.source            = { :git => 'https://github.com/Placenote/PlacenoteSDK-CocoaPod.git', :tag => 'v1.7' }
     s.swift_version     = '3.2'
```
Comment (nmathew87) [inline PlacenoteSDK.podspec]: does this need to be changed to 4 ? I remember we were having some issues with Swift version in the pod. Is this anything related?

File: PlacenoteSDK.podspec [hunk 3]
```diff
@@ -1,6 +1,6 @@
 Pod::Spec.new do |s|
     s.name              = 'PlacenoteSDK'
-    s.version           = '1.6.12'
+    s.version           = '1.7'
     s.summary           = 'Placenote SDK brings persistence to AR apps.'
```
Comment (nmathew87) [inline PlacenoteSDK.podspec]: Change this to - "Build apps that interact with the real world"

for consistency

File: Placenote.framework/Headers/libPlacenote.h [hunk 4]
```diff
@@ -114,6 +114,10 @@ FOUNDATION_EXPORT int PNSaveMap (const char* mapId, transfer_map_callback cb, vo
 FOUNDATION_EXPORT int PNLoadMap (const char* mapId, transfer_map_callback cb, void* context);
 FOUNDATION_EXPORT int PNDeleteMap (const char* mapId, result_callback cb, void* context);
 FOUNDATION_EXPORT int PNAddMap (result_callback cb, void* context);
+FOUNDATION_EXPORT int PNSyncThumbnail(const char* mapId, const char* thumbnailPath,
```
Comment (nmathew87) [inline Placenote.framework/Headers/libPlacenote.h:117]: Is this tested with the new version? The last update to the swift library was not tested with the archive that contains the thumbnail functions

File: Placenote.framework/_CodeSignature/CodeResources (hunk 1 above)
Comment (pmukherj) [reply Placenote.framework/_CodeSignature/CodeResources:13]: Yup. autogenerated. Was not added on this commit. Was always there.

File: Placenote.framework/Headers/libPlacenote.h (hunk 4 above)
Comment (pmukherj) [reply Placenote.framework/Headers/libPlacenote.h:117]: Yup. Its been tested.
